### PR TITLE
bringing dynatemp back

### PR DESCRIPTION
**Pull request by 50h100a**
like it never even left

### CODE REVIEW

File: aphrodite/modeling/sampling_metadata.py [hunk 1]
```diff
@@ -683,7 +720,7 @@ def from_lists(cls, temperatures: List[float], top_ps: List[float],
 
     @staticmethod
     def _get_sequence_seeds(
-        seed: int,
```
Comment (AlpinDale) [inline aphrodite/modeling/sampling_metadata.py:686]: What's the reason for this?

**Comment by AlpinDale** [comment]
Thanks for bringing it back, can you pipe it in the API server too?

File: aphrodite/modeling/sampling_metadata.py (hunk 1 above)
Comment (50h100a) [reply aphrodite/modeling/sampling_metadata.py:686]: It's just correcting the typehint.

`seed` comes from an `int|None` source, and this func has explicit behavior for the `None` case.

**Comment by 50h100a** [comment]
> Thanks for bringing it back, can you pipe it in the API server too?
Which? I tested it with the OpenAI endpoint, it seems to function there.

**Comment by AlpinDale** [comment]
> > Thanks for bringing it back, can you pipe it in the API server too?
> 
> Which? I tested it with the OpenAI endpoint, it seems to function there.

The API ignores any field that isn't present. You'll need to add it for the `ChatCompletionRequest` and `CompletionRequest` classes in the OAI server's procotol.

**Comment by 50h100a** [comment]
No wonder it ran fine.

**Pull request closed without merging**